pkg/shed/field_uint64.go

```diff
@@ -67,16 +67,19 @@ func (f Uint64Field) Put(val uint64) (err error) {
 
 // PutInBatch stores a uint64 value in a batch
 // that can be saved later in the database.
-func (f Uint64Field) PutInBatch(batch *leveldb.Batch, val uint64) {
-	batch.Put(f.key, encodeUint64(val))
+func (f Uint64Field) PutInBatch(batch *badger.Txn, val uint64) {
+	err := batch.Set(f.key, encodeUint64(val))
+	if err != nil {
+		f.logger.Debugf("could not set uint64 value in PutInBatch. Error : %s", err.Error())
```

Comment:
The error is not handled!

This is very dangerous. We should return this error in order to stop any other actions on the batch. This can be cause of data inconstancies.

Reply:
Do not log error and return it.